docs/userguide.md

```diff
@@ -247,6 +247,72 @@ done
 If you were able to verify the resources removed and added back then you have
 successfully verified a working federation-v2 deployment.
 
+In addition to specifying an explicit list of clusters that a resource should be
```

Comment:
I think a section header is suggested here to differentiate from the example above.

Reply:
Done